src/embedding/kd.ts

```diff
@@ -186,7 +186,20 @@ export namespace EmbeddedKpiDashboard {
         /**
          * Type represent that the remove filter context action is finished
          */
-        RemoveFilterContextFinished = 'removeFilterContextFinished'
+        RemoveFilterContextFinished = 'removeFilterContextFinished',
+
+        /**
+         * Type that represents the drill to url was started and loading can be displayed. It does not contain url.
+         * Url is contained by DrillToUrlResolved which is posted after url is resolved. Event also contains ID which
+         * can be then matched with DrillToUrlResolved ID
+         */
+        DrillToUrlStarted = 'drillToUrlStarted',
+
+        /**
+         * Type that represents the drill to url was resolved after DrillToUrlStarted and contains resolved url. Event
+         * also contains ID which can be matched with DrillToUrlStarted ID
+         */
```

Comment:
Same here:

"Type that represents resolved drill to URL. The event is sent after DrillToUrlStarted event was posted and it contains the resolved URL. The event also contains an ID which can be matched with ID from DrillToUrlStarted event."